setup/baseConfig.yaml

```diff
@@ -28,6 +28,7 @@ block_cache:
   disk-timeout-sec: <default disk cache eviction timeout (in sec). Default - 120 sec>
   prefetch: <number of blocks to be prefetched in serial read case. Min - 11, Default - 2 times number of CPU cores>
   parallelism: <number of parallel threads downloading the data and writing to disk cache. Default - 3 times number of CPU cores> 
+  use-du: <if true, du command will be used to calculate disk usage. Default - true>
```

Comment:
as a thumb rule, we have all bool flags set to false by default. This is breaking that rule so ideally we shall have a flag called "no-du" or "du-not-present" or something.